plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageBuildAction.java

```diff
@@ -343,6 +385,39 @@ public boolean hasDelta(final Baseline baseline, final Metric metric) {
         throw new NoSuchElementException("No such baseline: " + baseline);
     }
 
+    /**
+     * Returns whether a value for the specified metric exists.
+     *
+     * @param baseline
+     *         the baseline to use
+     * @param metric
+     *         the metric to check
+     *
+     * @return {@code true} if a value is available for the specified metric, {@code false} otherwise
+     */
+    public boolean hasValue(final Baseline baseline, final Metric metric) {
+        return getAllValues(baseline).stream()
+                .anyMatch(v -> v.getMetric() == metric);
+    }
+
+    /**
+     * Returns a formatted and localized String representation of the value for the specified metric (with respect to
+     * the given baseline).
+     *
+     * @param baseline
+     *         the baseline to use
+     * @param metric
+     *         the metric to get the delta for
+     *
+     * @return the formatted value
+     */
+    public String formatValue(final Baseline baseline, final Metric metric) {
+        var value = getAllValues(baseline).stream()
+                .filter(v -> v.getMetric() == metric)
+                .findFirst();
+        return value.isPresent() ? FORMATTER.formatValue(value.get()) : Messages.Coverage_Not_Available();
```

Comment:
This should be actually handled by the formatter

Reply:
It added this regarding to formatDelta(...) directly below. What is your suggestion here? Moving both methods to the formatter?

Reply:
No, seems that I looked at the wrong class. This is fine here.